conda.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+conda install -y \
+  python=3.6 \
```

Comment:
solvcon and the associated projects should be upgraded to the latest Python releases, in the future.

Reply:
I got this idea and +1 to it. We may add option to choose python version in the future. I did not change anything for this option in this pull request this time.

Reply:
Thanks.  Comments like this are useful.  I saw your smiley but smileys are hard to track.